subsys/rtio/rtio_handlers.c

```diff
@@ -70,9 +70,8 @@ static inline int z_vrfy_rtio_cqe_get_mempool_buffer(const struct rtio *r, struc
 }
 #include <syscalls/rtio_cqe_get_mempool_buffer_mrsh.c>
 
-static inline int z_vrfy_rtio_sqe_copy_in(struct rtio *r,
-					  const struct rtio_sqe *sqes,
-					  size_t sqe_count)
+static inline int z_vrfy_rtio_sqe_copy_in_get_handles(struct rtio *r, const struct rtio_sqe *sqes,
```

Comment:
If there's no other sqe_copy_in without handles, then this can be named rtio_sqe_copy_in as it was before shortening the name

Reply:
Discussed offline, leaving both signatures for now.